en/lessons/libraries/stream-data.md

```diff
@@ -0,0 +1,397 @@
+---
+version: 1.0.0
+title: StreamData
+---
+
+An example-based unit testing library like [ExUnit](https://hexdocs.pm/ex_unit/ExUnit.html) is a wonderful tool to help you verify that your code works the way you think it does. However, example-based unit tests have some drawbacks:
+
+* It can be easy to miss edge cases, since you're only testing a few inputs.
+* You can write these tests without thinking through your requirements thoroughly.
+* These tests can be very verbose when you use several examples for one function.
+
+In this lesson we're going to explore how [StreamData](https://github.com/whatyouhide/stream_data) can help overcome some of these drawbacks.
+
+
+{% include toc.html %}
+
+
+## What is StreamData?
+
+[StreamData](https://github.com/whatyouhide/stream_data) is library that performs stateless property-based testing.
```

Comment:
is a library

Reply:
Good catch! Those indefinite articles are important :grimacing: